### DISCUSSION TITLE
Add a filter for public squads, so you can find squads who focus on a specific topic.

### DISCUSSION THREAD

**Discussion by jalenlane7684** [dailydotdev/daily#1100] [Feature Requests]
I know there are not that many at the moment, but it would be a nice feature in the future. For example, if I want to find a JavaScript squad, instead of having to scroll through them, I should be able to either search for it or filter by certain tags.

**Comment by idoshamun** [comment]
Hi @jalenlane7684,

Thank you for the suggestion!!

**Comment by sabdailydev** [comment]
We have several categories now on Public Squads. This was launched several weeks back. [Here](https://app.daily.dev/squads/discover)

**Reply by jalenlane7684** [reply]
It looks awesome!
